### DISCUSSION TITLE
Group access vs successive access

### DISCUSSION THREAD

**Discussion by jmachta** [Koenkk/zigbee2mqtt#16073] [General]
Guys, I have a question about what is the actual difference from zigbee perspective when I bind the devices into group and control them as a group vs. when I do control them just one-by-one. Example: In living room I have a ceiling light with three bulbs. I use openhab to control these bulbs, having each of them as generic mqtt thing. Then when controlling this light, I need to send three mqtt commands to change brightness. 

If I bound them into group, I would issue just one mqtt command for whole group. Of course, I save 2 mqtt commands. But from zigbee perspective, does it mean that each bulb still receives a separate command? In other words, does the grouping happen only on level of z2m and means nothing for zigbee itself or is there a different command type sent to zigbee network? e.g some type of broadcast? 

Thank you a lot for explanation...

**Comment by lux73** [comment]
i had a same Setup with two Bulbs accessed via OpenHAB

before i group both Channels via .items File:

```
Dimmer Hue_Bad_dimmer	        "Hue Bad Dimmer [%d %%]"	<slider>	(gBad,gHueDim,gHueDimBad) {channel="mqtt:topic:hue_bad_rechts:helligkeit", channel="mqtt:topic:hue_bad_links:helligkeit", autoupdate="true"}
Dimmer Hue_Bad_colortemp	"Hue Bad Farbtemperatur [%d]"	<colorlight>	(gBad)			  {channel="mqtt:topic:hue_bad_rechts:colortemp", channel="mqtt:topic:hue_bad_links:colortemp", autoupdate="true"}
```
but now i using Group via z2m

afaik Group Commands via z2m are realized via Broadcast - so my z2m Webinterface "last seen" don't work anymore as expected

and i had to slow down my Rule Commands after grouping via z2m between dimmer & colortemp changes because OpenHAB sends the commands to fast - the second command go missing...

`Thread::sleep(70)` between dimmer & colortemp changes will do

**Comment by jmachta** [comment]
Thank you a lot for sharing your experience. Based on it I tried this approach where possible in my installation and it seems to have some positive effect on behavior - reactions of some devices are faster. Also my scripts got a little bit simpler, which is always good.
